tests/tr1/tests/cstdio/test.cpp

```diff
@@ -126,13 +128,15 @@ void test_cpp() { // test C++ header
     { // test character I/O
         const char* tmpbuff;
         char tname[L_tmpnam];
-        char tmpbuf[L_tmpnam];
+        const char* tmpbuf;
         STDx FILE* pf;
         char tn[100] = {0};
 
-        assert(STDx tmpnam(tmpbuf) == tmpbuf);
+        const auto temp_name1 = temp_file_name();
+        assert((tmpbuf = temp_name1.c_str()) != nullptr);
         CHECK(CSTD strlen(tmpbuf) < L_tmpnam);
-        assert((tmpbuff = STDx tmpnam((char*) nullptr)) != nullptr);
+        const auto temp_name2 = temp_file_name();
+        assert((tmpbuff = temp_name2.c_str()) != nullptr);

```

Comment:
I know you're trying to keep the changes minimal, but this is the third `assert` validating invariants of `std::string` which isn't the component we're testing here.

Reply:
You're right, this makes no sense now. Removing.